numpy/linalg/umath_linalg.c.src

```diff
@@ -706,10 +706,21 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
-#ifndef min
-#define min(x,y) ((x<y)?x:y)
-#define max(x,y) ((x>y)?x:y)
-#endif
+static inline fortran_int
```

Comment:
Hmm, that is strange. Then again, the npy version was needed when not all compilers supported the C99 standard and that may no longer be a problem.